### PR TITLE
Introduce separate line width style for tab bends

### PR DESCRIPTION
**Pull request by mike-spa**
...and change the Bend style menu accordingly.

### CODE REVIEW

**Review by oktophonie — APPROVED** [review]

File: src/notation/view/widgets/editstyle.ui [hunk 1]
```diff
@@ -8993,125 +8993,173 @@ By default, they will be placed such as that their right end are at the same lev
...
+                  <width>40</width>
+                  <height>20</height>
+                 </size>
+                </property>
+               </spacer>
+              </item>
+              <item row="0" column="0">
+               <widget class="QLabel" name="label_171">
+                <property name="text">
+                 <string>Lien thickness: </string>
```
Comment (oktophonie) [inline src/notation/view/widgets/editstyle.ui:9019]: Line (typo)

**Review by cbjeukendrup — APPROVED** [review]

**Review by bkunda — CHANGES_REQUESTED** [review]
Pls just adjust the spin box for Standard Stave Line Thickness to adjust in 0.1 increments, rather than 1.0 [See video](https://www.dropbox.com/s/4j3obdt2ighcn15/line-thickness-settings.mov?dl=0)

**Comment by mike-spa** [comment]
I think 0.1 is still way too coarse (a single +- 0.1 click already brings the values into unreasonable territory). So I actually made it 0.01, which feels better (and the reasonable territory is just few clicks away in either direction)

**Review by bkunda — APPROVED** [review]
Tested on macOS. FIXED:

**Pull request merged**